DuckDuckGo/FavoritesOverlay.swift

```diff
@@ -40,11 +42,16 @@ class FavoritesOverlay: UIViewController {
     
     private var theme: Theme!
     
+    fileprivate var viewModelCancellable: AnyCancellable?
```

Comment:
Minor nit - is there any reason why this can't be made `private` instead?

Reply:
Fixed, thanks.  Not really..  Just follow the pattern I saw in other bookmark mgmt places, but also not likely needed there.